app/src/main/java/org/simple/clinic/bp/assignbppassport/BpPassportEvent.kt

```diff
@@ -1,3 +1,5 @@
 package org.simple.clinic.bp.assignbppassport
 
 sealed class BpPassportEvent
+
+data class RegisterNewPatientClicked(val bpPassportNumber: Int) : BpPassportEvent()
```

Comment:
Instead of passing the BP passport identifier with the event. We can save it in the model during screen init, since we will have to BP passport identifier when opening the screen. WDYT?